projects/packages/paypal-payments/src/legacy/class-simple-payments.php

```diff
@@ -0,0 +1,834 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Simple Payments lets users embed a PayPal button fully integrated with wpcom to sell products on the site.
+ * This is not a proper module yet, because not all the pieces are in place. Until everything is shipped, it can be turned
+ * into module that can be enabled/disabled.
+ *
+ * @package automattic/jetpack-paypal-payments
+ */
+
+namespace Automattic\Jetpack\Paypal_Payments;
+
+use Automattic\Jetpack\Assets;
+use Automattic\Jetpack\Current_Plan as Jetpack_Plan;
+use Jetpack;
+use Jetpack_Components;
+use Jetpack_Currencies;
+use Jetpack_Options;
+use WP_Post;
+
+/**
+ * Simple_Payments
+ */
+class Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
```

Comment:
This seems to be a left-over from something else.

Reply:
It was. I forgot to remove it. Removed in [83ee3d0](https://github.com/Automattic/jetpack/pull/43413/commits/83ee3d07174550f9d18b4a8654f5644b20fe66f4).